src/main/kotlin/week1/racingcar/ResultView.kt

```diff
@@ -0,0 +1,18 @@
+package week1.racingcar
+
+class ResultView {
+
+    init {
+        println("실행 결과")
+    }
+
+    fun showResult(cars: List<Car>) {
+        cars.forEach {
+            (0 until it.mileage).forEach { _ ->
+                print("-")
+            }
+            println()
+        }
```

Comment:
repeat 함수를 활용해서 더 간결한 코드로 변경해볼 수 있을 것 같아요. 😃
```suggestion
        cars.forEach { println("-".repeat(it.mileage)) }
```